app/src/main/java/org/mozilla/focus/browser/BrowserNavigationOverlay.kt

```diff
@@ -68,6 +70,16 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
      */
     private var uiLifecycleCancelJob: Job
 
+    /**
```

Comment:
nit: `/**` comments are used to describe to the caller what the function/variable does when they use it. `//` (repeated when multiple lines) is used to describe implementation details. In this case, we're describing the latter!